site-src/api-types/httproute.md

```diff
@@ -85,14 +85,16 @@ Take the following matches configuration as an example:
 apiVersion: gateway.networking.k8s.io/v1beta1
 kind: HTTPRoute
 ...
-matches:
-  - path:
-      value: "/foo"
-    headers:
-      values:
-        version: "2"
-  - path:
-      value: "/v2/foo"
+spec:
+  rules:
+  - matches:
+    - path:
+        value: "/foo"
+      headers:
+        values:
```

Comment:
header match should be a list with at least header name and value as explicit fields (not a map like here), see https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1beta1.HTTPHeaderMatch